### PR TITLE
Enable elastic scaling node side feature on Polkadot 

### PR DESCRIPTION
**Pull request by sandreim**
This feature is needed on Polkadot to ensure parachains can still make progress if they have multiple cores assigned. Relay chain support for elastic scaling is implemented but requires bumping `polkadot-runtime-parachains` crate version to at least `11.0.0` which will happen later.

- [ ] Does not require a CHANGELOG entry

### CODE REVIEW

**Review by alindima — APPROVED** [review]

File: relay/polkadot/src/lib.rs
Comment (eskimor) [inline relay/polkadot/src/lib.rs:1899]: Why are we not using the [constant](https://github.com/paritytech/polkadot-sdk/blob/523e62560eb5d9a36ea75851f2fb15b9d7993f01/polkadot/primitives/src/v7/mod.rs#L2036)?

Comment (sandreim) [reply relay/polkadot/src/lib.rs:1899]: It's currently in staging, felt that we don't want to use any of the staging stuff here.

Comment (eskimor) [reply relay/polkadot/src/lib.rs:1899]: Hmm - don't we have v7 already in the current release? In any case, seems like we would be using staging stuff anyways, just hiding it by using a magic number? Also I think this can wait for the next polkadot-sdk release, where this should be v7 then.

Comment (sandreim) [reply relay/polkadot/src/lib.rs:1899]: That makes sense, but this needs to go in same release with core time.

**Comment by eskimor** [comment]
@sandreim Let's rebase on [this](https://github.com/polkadot-fellows/runtimes/pull/322) one.

**Comment by eskimor** [comment]
> @sandreim Let's rebase on [this](https://github.com/polkadot-fellows/runtimes/pull/322) one.

Actually, ignore. I can just make this part of my coretime enabling PR while I am already at it.

**Review by bkchr — CHANGES_REQUESTED** [review]
Do we again try to set some configuration option, that could just be a separate referendum? :P 

Please close and just open a referendum on chain.

**Comment by alindima** [comment]
> Do we again try to set some configuration option, that could just be a separate referendum? :P
> 
> Please close and just open a referendum on chain.

I think the reasoning of doing this as a migration was that it's more of a bugfix than a new feature. Once coretime is enabled on polkadot, if this feature bit is not set, one para that gets scheduled on two cores will brick itself.

**Comment by bkchr** [comment]
> I think the reasoning of doing this as a migration was that it's more of a bugfix than a new feature.

This can already be opened today and ready before Coretime lands on Polkadot. Not really an argument to put this into a runtime upgrade.

We have done it [here](https://github.com/polkadot-fellows/runtimes/pull/205) using a runtime upgrade, because the feature was added in the same release AFAIR. However, this is not the case anymore.

**Comment by sandreim** [comment]
https://polkadot.subsquare.io/referenda/1007

**Pull request closed without merging**
